lighthouse-core/fraggle-rock/config/default-config.js

```diff
@@ -7,6 +7,29 @@
 
 const legacyDefaultConfig = require('../../config/default-config.js');
 
+/** @type {LH.Config.AuditJson[]} */
+const frAudits = [
+  'byte-efficiency/uses-responsive-images-snapshot',
+];
+
+/** @type {Record<string, LH.Config.AuditRefJson[]>} */
+const frCategories = {
```

Comment:
WDYT about making this an actual category object?

Reply:
The reason I didn't do this originally was because the category requires a title, and it didn't make sense to add that if we are just goin to extract the category contents. This isn't really a category object, it's just additional contents for the one used on the default config.

Reply:
gotcha, that's fair.

```suggestion
const frCategoryAuditRefExtensions = {
```

or something? :)

gotta make our variable names useful training data for copilot 🤣